stable/vpa/templates/admission-controller-service.yaml

```
@@ -2,12 +2,12 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "vpa.fullname" . }}-admission-controller
+  name: vpa-webhook
 spec:
   ports:
     - port: 443
       targetPort: 8000
   selector:
     app.kubernetes.io/component: admission-controller
-    {{- include "vpa.labels" . | nindent 4 }}
+    {{- include "vpa.selectorLabels" . | nindent 4 }}
```

I just noticed that this will currently select the other pods from the recommender and the updater. We need to add in the `app.kubernetes.io/component: admission-controller` label as well.

```suggestion
    app.kubernetes.io/component: admission-controller
    {{- include "vpa.selectorLabels" . | nindent 4 }}
```